File: tmdt-ui/angular/src/app/_models/Topic.ts
export class Topic {
  private topicID: string;
  private topicName: string;
  private topicDescription: string;
  private topicStatus: number;

  constructor(
    topicID: string,
    topicName: string,
    topicDescription: string,
    topicStatus: number
  ) {
    this.topicID = topicID;
    this.topicName = topicName;
    this.topicDescription = topicDescription;
    this.topicStatus = topicStatus;
  }
  public get $topicID(): string {
    return this.topicID;
  }

  public set $topicID(value: string) {
    this.topicID = value;
  }

  public get $topicDescription(): string {
    return this.topicDescription;
  }

  public set $topicDescription(value: string) {
    this.topicDescription = value;
  }

  public get $topicStatus(): number {
    return this.topicStatus;
  }

  public set $topicStatus(value: number) {
    this.topicStatus = value;
  }

  public get $topicName(): string {
    return this.topicName;
  }

  public set $topicName(value: string) {
    this.topicName = value;
  }
}

File: tmdt/src/main/java/com/spring/domain/Topic.java
package com.spring.domain;

public class Topic {
	private String topicID;
	private String topicName;
	private	String topicDescription;
	private Integer topicStatus;
	public Topic(String topicID, String topicName, String topicDescription, Integer topicStatus) {
		super();
		this.topicID = topicID;
		this.topicName = topicName;
		this.topicDescription = topicDescription;
		this.topicStatus = topicStatus;
	}
	public Topic() {
		super();
	}
	public String getTopicID() {
		return topicID;
	}
	public void setTopicID(String topicID) {
		this.topicID = topicID;
	}
	public String getTopicName() {
		return topicName;
	}
	public void setTopicName(String topicName) {
		this.topicName = topicName;
	}
	public String getTopicDescription() {
		return topicDescription;
	}
	public void setTopicDescription(String topicDescription) {
		this.topicDescription = topicDescription;
	}
	public Integer getTopicStatus() {
		return topicStatus;
	}
	public void setTopicStatus(Integer topicStatus) {
		this.topicStatus = topicStatus;
	}
	@Override
	public String toString() {
		return "Topic [topicID=" + topicID + ", topicName=" + topicName + ", topicDescription=" + topicDescription
				+ ", topicStatus=" + topicStatus + "]";
	}
	
	
}
